less/kikar.less

```diff
@@ -25,4 +25,72 @@
 #loading-statuses-symbol{
   text-align: center;
   display: none;
-}
\ No newline at end of file
+}
+
+
+/*  Styling for responsive embedding by http://embedresponsively.com/*/
+
+.embed-container {
+    position: relative;
+    padding-bottom: 56.25%;
+    height: 0;
+    overflow: hidden;
+    max-width: 100%;
+}
+
+.embed-container iframe, .embed-container object, .embed-container embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
+ /*  Styles for _facebook_status_section_update_elem.html */
+.fb-attachment-container {
+    text-align: center;
+    border-style: solid none none none;
+    border-top-color: #dddddd;
+    border-top-width: thin;
+}
+
+.img-fb-status-attachment {
+    height: auto;
+    margin: auto;
+    border: solid;
+    border-width: thin;
+    border-color: #dddddd;
+}
+
+@media (min-width: 768px) {
```

Comment:
I am not sure I understand the second comment. I'm compiling all changes made in `less` files.